install/linux/usr/share/odemis/hwtest/tmcm-can-ctrl.odm.yaml

```diff
@@ -0,0 +1,20 @@
+SimOptical: {
+    class: Microscope,
+    role: brightfield,
+    children: [Light, TMCM_CAN],
+}
+
+Light: {
+    class: simulated.Light,
+    role: brightlight,
+}
+
+TMCM_CAN: {
```

Comment:
Let's call this component with an easier name to spell "TMCMCan" would be OK, or maybe "Rotator".